### ISSUE TITLE
impt fields too small

### ISSUE THREAD

**Issue by swyxio** [sw-yx/bettertwitter#2]
the search field is the most important field in this entire form and is too small right now

![image](https://user-images.githubusercontent.com/6764957/71308621-fb060e00-23cc-11ea-9f93-741f8fb3171d.png)

i want to be easily able to find it. @wgao19 got ideas?

**Comment by wgao19** [comment]
Maybe
- give it its own section
- make the input bar full width and taller
- move to top of the page?

**Comment by swyxio** [comment]
yeah. up to u.. u can see what i ended up doing with the svelte rewrite
